metrics/README.md

```diff
@@ -0,0 +1,17 @@
+## Measuring Success
+
+It's important to think about what success means to you. Have clear goals before getting started, including a plan on how to measure. 
+
+### Could you measure...
```

Comment:
`

formula_install() {
--
  | PACINSTALL=false
  | for i in "${@}"; do
  | if ! command -v "${i}" > /dev/null 2>&1
  | then
  | echo -e "Installing ${i}"
  | $PACMAN "${i}" && PACINSTALL=true
  | fi
  | done
  | if [ "${PACINSTALL}" == true ]
  | then
  | return 0
  | else
  | return 1
  | fi
  | }

`